tslint.json

```diff
@@ -0,0 +1,61 @@
+
+{
```

Comment:
👉 Let's use https://github.com/dcos-labs/tslint-config instead of rolling yet another config.

Reply:
Completely forgot about it, good catch 👍